salt/modules/debbuild.py

```diff
@@ -264,6 +273,14 @@ def _create_pbuilders(env):
     if env_overrides and not env_overrides.isspace():
         with salt.utils.files.fopen(pbuilderrc, 'a') as fow:
             fow.write(salt.utils.stringutils.to_str(env_overrides))
+    cmd = "chown {0}:{0} {1}".format(runas, pbuilderrc)
```

Comment:
Is there a reason that `file.chown` isn't being used here instead?

Reply:
Went looking for it and missed it in the doc's, will make change

Reply:
Ah, OK. That makes sense.